gradle.properties

```diff
@@ -1,15 +1,14 @@
 #org.gradle.console=rich
 org.gradle.parallel=true
 org.gradle.workers.max=32
-
 javaVersion=11
 infoModelVersion=4.2.7
 jetBrainsAnnotationsVersion=15.0
 jacksonVersion=2.13.1
 jettyVersion=11.0.6
 websocketVersion=2.0.0
 jerseyVersion=3.0.3
-servletApi=4.0.4
+servletApi=4.0.1
```

Comment:
Could you explain in the PR the reason for this downgrade?

Reply:
Restored as in main branch.